citrination_client/client.py

```diff
@@ -173,6 +183,33 @@ def _get_create_data_set_url(self):
         """
         return self.api_url+'/data_sets/create_dataset'
 
+    def update_data_set(self, data_set_id, name=None, description=None, share=None):
+        """
+        Update a data set.
+        :param data_set_id:
+        :param name: name of the dataset
+        :param description: description for the dataset
+        :param share: share the dataset with everyone on the site. Valid values are '1' or '0'.
```

Comment:
Can you be very explicit here that 1 means that it is public and 0 means that it is not.